thrift/events.thrift

```diff
@@ -8,15 +10,15 @@ struct ContentAtomEvent {
    * across all content-atoms of any type (an alternative model might
    * be to have the `type+id` be unique) */
 
-   1: ContentAtomID id
+   1: required ContentAtomID id
 
    // this is a canonical place from which the current version of the
    //content atom's data can be downloaded.
-   2: string url
+   2: required string url
```

Comment:
>It might not be needed you are right. But this is connected only to the event message, so although it might change, it is only intended to be correct for the lifetime of the event message, which in reality would be measured in minutes if not less.

 *  The url of the atom may not be known by the content producer.
 *  If we send those events to a stream, the `lifetime` of the event message